server/api/playbooks.go

```diff
@@ -91,6 +91,20 @@ func (h *PlaybookHandler) createPlaybook(w http.ResponseWriter, r *http.Request)
 		}
 	}
 
+	if pbook.WebhookOnStatusUpdateURL != "" {
```

Comment:
We should change this to the following, so we can actually save playbooks that have the setting disabled, even if the URL is wrong.

```suggestion
	if pbook.WebhookOnStatusUpdateEnabled {
```